AUTHORS

```diff
@@ -1,3 +1,6 @@
 Timur Shemsedinov <timur.shemsedinov@gmail.com>
+Dimon Durak <Dimon Durak>
+Artem Chernenkiy <notthewhite@gmail.com>
 Alexey Orlenko <eaglexrlnk@gmail.com>
 Mykola Bilochub <nbelochub@gmail.com>
+Vlad <Dzyubavlad@gmail.com>
```

Comment:
@aqrln, okey. Right spelling of my last name is Dziuba. Let it be this way. My email case does not matter, so you can do what it is needed with it.